althea/althea-rust-binaries/files/rita.toml

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh /etc/rc.common
```

Comment:
same for this

Reply:
this is for the creation of a service, half finished, I'll go wrap that up before merging this.